src/main/java/com/epam/model/service/TextFileService.java

```diff
@@ -0,0 +1,110 @@
+package com.epam.model.service;
+
+import com.epam.model.entity.TextInfoHolder;
+
+import java.io.*;
+import java.util.StringTokenizer;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+import com.epam.model.entity.TextInfoHolder.Builder;
+
+public class TextFileService {
```

Comment:
Add private constructor to avoid instance creation. Utility classes, which are collections of static members, are not meant to be instantiated.